geomesa-spark/geomesa-spark-converter/src/test/scala/org/locationtech/geomesa/spark/converter/ConverterSpatialRDDProviderTest.scala

```diff
@@ -0,0 +1,61 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.spark.converter
+
+import com.typesafe.config.{ConfigFactory, ConfigRenderOptions}
+import org.apache.hadoop.conf.Configuration
+import org.apache.spark.{SparkConf, SparkContext}
+import org.geotools.data.Query
+import org.geotools.filter.text.ecql.ECQL
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.spark.{GeoMesaSpark, GeoMesaSparkKryoRegistrator}
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import scala.collection.JavaConversions._
+
+@RunWith(classOf[JUnitRunner])
+class ConverterSpatialRDDProviderTest extends Specification {
+
+  import ConverterSpatialRDDProvider.{ConverterKey, InputFilesKey, SftKey}
+
+  var sc: SparkContext = null
+
+  step {
+    val conf = new SparkConf().setMaster("local[2]").setAppName("testSpark")
+    conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+    conf.set("spark.kryo.registrator", classOf[GeoMesaSparkKryoRegistrator].getName)
+    sc = SparkContext.getOrCreate(conf)
+  }
+
+  val exampleConf = ConfigFactory.load("example.conf")
+  val converterConf = exampleConf.getConfig("geomesa.converters.example-csv")
+
+  val params = Map(
+    InputFilesKey -> getClass.getResource("/example.csv").getPath,
+    ConverterKey  -> converterConf.root().render(ConfigRenderOptions.concise()),
+    SftKey        -> exampleConf.root().render(ConfigRenderOptions.concise())
```

Comment:
Maybe put the actual key strings here so that the test can serve as a self-contained example.